docs/source/daml/intro/7_Composing.rst

```diff
@@ -0,0 +1,271 @@
+
+          #15:6
+          │   referenced by #17:0
+          │   known to (since): 'Bob' (#15), 'USD_Bank' (#15), 'Alice' (#15)
+          └─> create Intro.Asset:Asset
+              with
+                issuer = 'USD_Bank'; owner = 'Bob'; symbol = "USD"; quantity = 100.0; observers = []
+
+      #15:7
+      │   known to (since): 'EUR_Bank' (#15), 'Alice' (#15), 'Bob' (#15)
+      └─> 'Bob',
+          'Alice' exercises TransferApproval_Transfer on #12:1 (Intro.Asset:TransferApproval)
+                  with
+                    assetCid = #10:1
+          children:
+          #15:8
+          │   known to (since): 'EUR_Bank' (#15), 'Alice' (#15), 'Bob' (#15)
+          └─> fetch #10:1 (Intro.Asset:Asset)
+
+          #15:9
+          │   known to (since): 'Bob' (#15), 'EUR_Bank' (#15), 'Alice' (#15)
+          └─> 'Bob', 'EUR_Bank' exercises Archive on #10:1 (Intro.Asset:Asset)
+
+          #15:10
+          │   referenced by #16:0
+          │   known to (since): 'Alice' (#15), 'EUR_Bank' (#15), 'Bob' (#15)
+          └─> create Intro.Asset:Asset
+              with
+                issuer = 'EUR_Bank'; owner = 'Alice'; symbol = "EUR"; quantity = 90.0; observers = []
+
+Similar to choices, you can see how the scenarios in this project are built up from each other:
+
+.. literalinclude:: daml/daml-intro-7/daml/Test/Intro/Asset/Role.daml
+  :language: daml
+  :start-after: -- TEST_ISSUANCE_BEGIN
+  :end-before: -- TEST_ISSUANCE_END
+
+In the above, the ``test_issuance`` scenario in ``Test.Intro.Asset.Role`` uses the output of the ``setupRoles`` scenario in the same module.
+
+The same line shows a new kind of pattern matching. Rather than writing ``setupResults <- setupRoles`` and then accessing the components of ``setupResults`` using ``_1``, ``_2``, etc., you can give them names. It's equivelent to writing
+
+.. code-block:: daml
+
+  setupResults <- setupRoles
+  case setupResults of
+    (alice, bob, bank, aha, ahb) -> ...
+
+Just writing ``(alice, bob, bank, aha, ahb) <- setupRoles`` would also be legal, but ``setupResults`` is used in the return value of ``test_issuance`` so it makes sense to give it a name, too. The notation with ``@`` allows you to give both the whole value as well as its constituents names in one go.
+
+.. _execution_model:
+
+DAML's execution model
+----------------------
+
+DAML's execution model is fairly easy to understand, but has some important consequences. You can imagine the life of a transaction as follows:
+
+1. A party submits a transaction. Remember, a transaction is just a list of actions.
+2. The transaction is interpreted, meaning the ``Update`` corresponding to each action is evaluated in the context of the ledger to calculate all consequeces, including transitive ones (consequences of consequences, etc.)
+3. The transaction is *blinded*, or *projected*, meaning the views that different parties have on the transaction are calculated.
+4. The blinded views are distributed to the parties.
```

Comment:
What does that mean? (I don't think you've touched on anything that would explain this so far)

Reply:
What does what mean? I say what "blinded" means in item 3.